src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -90,23 +81,15 @@ def link_user(self, slack_request: SlackDMRequest) -> Response:
             response_url=slack_request.response_url,
         )
 
-        metrics.incr(self._METRICS_SUCCESS_KEY + ".link_user", sample_rate=1.0)
-
         return self.reply(slack_request, LINK_USER_MESSAGE.format(associate_url=associate_url))
 
     def unlink_user(self, slack_request: SlackDMRequest) -> Response:
         from sentry.integrations.slack.views.unlink_identity import build_unlinking_url
 
         if not slack_request.has_identity:
-            logger.error(".unlink-user.no-identity.error", extra={"slack_request": slack_request})
-            metrics.incr(self._METRIC_FAILURE_KEY + "unlink_user.no_identity", sample_rate=1.0)
-
             return self.reply(slack_request, NOT_LINKED_MESSAGE)
 
         if not (slack_request.integration and slack_request.user_id and slack_request.channel_id):
-            logger.error(".unlink-user.bad_request.error", extra={"slack_request": slack_request})
-            metrics.incr(self._METRIC_FAILURE_KEY + "unlink_user.bad_request", sample_rate=1.0)
```

Comment:
Let's update our webhook metrics dashboards for webhooks before merging this please!